homeassistant/components/browan/config_flow.py

```diff
@@ -0,0 +1,110 @@
+"""Config flow for Browan integration."""
+from __future__ import annotations
+
+import logging
+import re
+from typing import Any
+
+from pyliblorawan.helpers.exceptions import (
+    CannotConnect,
+    DeviceEuiNotFound,
+    InvalidAuth,
+    InvalidDeviceEui,
+)
+from pyliblorawan.models import Device
+from pyliblorawan.network_servers.ttn import TTN
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_MODEL, CONF_URL
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.selector import (
+    SelectOptionDict,
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+)
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+_RE_DEVICE_EUI = re.compile(r"^[0-9a-fA-F]{16}$")
+
+_DEVICE_SELECTOR = SelectSelector(
+    SelectSelectorConfig(
+        options=[SelectOptionDict(value="TBMS100", label="TBMS100")],
+        mode=SelectSelectorMode.DROPDOWN,
+    )
+)
+
+_STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_URL, default="https://eu1.cloud.thethings.network"): str,
+        vol.Required("application"): str,
+        vol.Required("api_key"): str,
```

Comment:
we should probably use consts for all schema keys here, api_key can be imported from homeassistant.const

Reply:
Ok, fixed in 798765188e571474659bedce020c66872d095b5e